pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -329,6 +344,22 @@ func GatherClusterAuthentication(i *Gatherer) func() ([]record.Record, []error)
 	}
 }
 
+// GatherClusterImageRegistry fetches the cluster Image Registry configuration
```

Comment:
Could you please also run `make gen-doc` ? That should update the `doc/gathered-data.md` with this new method.

Reply:
done